pkg/operator/status.go

```diff
@@ -38,18 +38,20 @@ func (optr *Operator) statusProgressing() error {
 		glog.Errorf("error getting current versions: %v", err)
 		return err
 	}
-
+	var isProgressing osconfigv1.ConditionStatus
 	var message string
 	if !reflect.DeepEqual(desiredVersions, currentVersions) {
 		message = fmt.Sprintf("Progressing towards %s", optr.printOperandVersions())
+		isProgressing = osconfigv1.ConditionTrue
 	} else {
 		message = fmt.Sprintf("Running resync for %s", optr.printOperandVersions())
+		isProgressing = osconfigv1.ConditionFalse
 	}
 
 	conds := []osconfigv1.ClusterOperatorStatusCondition{
 		{
 			Type:    osconfigv1.OperatorProgressing,
-			Status:  osconfigv1.ConditionTrue,
+			Status:  isProgressing,
```

Comment:
Looks good :) can you also add a test here [1] to verify the condition status is true when `desiredVersions` != `currentVersions` and false when both are equal?

[1] https://github.com/openshift/machine-api-operator/blob/master/pkg/operator/status_test.go

Reply:
+1, also including `failing=false` and `progressing=false` here would be nice https://github.com/openshift/machine-api-operator/blob/master/test/e2e/operator_expectations.go#L68

Reply:
@ingvagabund done